app/assets/stylesheets/notes.scss

```diff
@@ -0,0 +1,77 @@
+table {
```

Comment:
Applying style to a `table` will work as long as there is only one table across the project. Consider using a class/ID for the table that is being styled.